README.md

```diff
@@ -5,13 +5,21 @@
 This is a documentation site for [Synapse](https://www.synapse.org). Synapse is an open source software platform that data
 scientists use to carry out, track, and communicate their research in real time.
 
-## How to contribute
+## Contributing Guide
 
-Synapse Docs is generated using [Jekyll](https://jekyllrb.com/) and uses redcarpet to render Markdown. Various page layouts can be found under the \_layouts folder in the home directory. Most everything can be written using standard [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).
+- Assign GitHub issue to yourself to track work in progress and prevent duplicate efforts. If you don't know what you should work on, look for things tagged with `help-wanted` or `good-first-issue`.
+- Create a [feature branch](https://guides.github.com/introduction/flow/) from the `gh-pages` branch to make your changes or new contribution.
+- Using a Markdown editor lke Typora, VS Code, or Sublime Text will allow you to visualize how your Markdown looks locally. You can also make your changes directly in the GitHub website or any other text editor.
+- To build locally, follow the instructions found [here](https://help.github.com/en/articles/testing-your-github-pages-site-locally-with-jekyll).
```

Comment:
```suggestion
```
We can remove this bullet since it is further explained below.

Reply:
Cool! I can't commit this, but you can! :D